core/urlproc.py

```diff
@@ -7,6 +7,13 @@
 from core import urlmarker
 
 
+def remove_empty(file_list):
+    """
+    Given a file list, return only those that aren't empty string or None.
```

Comment:
I really like to have a list of parameters, their types and the type and nature & the return according to the numpy style in the docstring. This is very useful later to generate automated docs ;)

Reply:
Sure I'll add this too.